cfme/test_requirements.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+
+alert = pytest.mark.requirement("alert")
```

Comment:
there is no markers module in the cfme namespace
the module name was picked for showing intent